packages/iris-grid/src/IrisGridModelUpdater.tsx

```diff
@@ -143,7 +143,8 @@ function IrisGridModelUpdater({
         model.rollupConfig = rollupConfig;
       }
     },
-    [model, rollupConfig]
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [model, model.rollupConfig, rollupConfig]
```

Comment:
This should be `model.isRollupAvailable`

I found out a way to not need the eslint exception. Just destructure the model to get `isRollupAvailable` as a separate variable and then use that instead of `model.isRollupAvailable`.

Weirdly, with `useEffect` it didn't complain with or without `model.isRollupAvailable`